src/Footer.tsx

```diff
@@ -205,6 +220,47 @@ export const Footer = memo(
                             </ul>
                         </div>
                     </div>
+                    {partnersLogos && (
+                        <div className="fr-footer__partners">
+                            <h4 className="fr-footer__partners-title">{t("our partners")}</h4>
+                            <div className="fr-footer__partners-logos">
+                                <div className="fr-footer__partners-main">
+                                    <a
+                                        className="fr-footer__partners-link"
+                                        href={partnersLogos.main.href}
+                                    >
```

Comment:
```suggestion
                                    <a
                                        className="fr-footer__partners-link"
                                        href={partnersLogos.main.href}
                                        taget="_blank"
                                    >
```
We probably want to open a new tab don't we?